emcc.py

```diff
@@ -1222,7 +1245,7 @@ def get_bitcode_args(input_files):
     # First, generate LLVM bitcode. For each input file, we get base.o with bitcode
     for i, input_file in input_files:
       file_ending = filename_type_ending(input_file)
-      if file_ending.endswith(SOURCE_ENDINGS):
+      if file_ending.endswith(SOURCE_ENDINGS) or has_fixed_language_mode:
```

Comment:
Why not? It's the same behavior clang has. If -x c++ or c is given clang treats all input file as c++/c source files. Am I missing here something?